pkg/controller/migmigration/quiesce.go

```diff
@@ -0,0 +1,78 @@
+package migmigration
+
+import (
+	"context"
+
+	appsv1 "github.com/openshift/api/apps/v1"
+	coreappsv1 "k8s.io/api/apps/v1"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// All quiesce functionality should be put here
+func (t *Task) quiesceApplications() error {
+	// If this is stage don't quiesce anything
+	if t.stage() {
+		return nil
+	}
+	err := t.scaleDownDCs()
+	if err != nil {
+		return err
+	}
+	err = t.scaleDownDeployments()
+	return err
+}
+
+// Scales down Deployment Configs on source cluster
+func (t *Task) scaleDownDCs() error {
+	client, err := t.getSourceClient()
+	if err != nil {
+		return err
+	}
+	for _, ns := range t.PlanResources.MigPlan.Spec.Namespaces {
+		list := appsv1.DeploymentConfigList{}
+		options := k8sclient.InNamespace(ns)
+		err = client.List(
+			context.TODO(),
+			options,
+			&list)
+		if err != nil {
+			return err
+		}
+		for _, dc := range list.Items {
+			dc.Spec.Replicas = 0
```

Comment:
We should only send an update call if replica count if it isn't already set to 0.